cli/cmd/auth/authenticator.go

```diff
@@ -0,0 +1,247 @@
+package auth
```

Comment:
Since it's not defining CLI commands, move this file to `cli/pkg/auth` (or maybe `deviceauth`?)